### DISCUSSION TITLE
v3.59.1.0 win64 cant run from network drive

### DISCUSSION THREAD

**Discussion by msmcknight** [mifi/lossless-cut#1991] [Support]
If I unzip the package to a local drive and run it, it works fine.  If I unzip the package to a network drive, it fails as soon as it starts.

I get the initial lossless-cut window frame, then it turns white and fails with the error:
LosslessCut has stopped working
A problem caused the program to stop working correctly.  Please close the program.

I don't have trouble running any other programs from this same network share.

The windows application logs show the failure as:
```
- System 

  - Provider 

   [ Name]  Application Error 
 
  - EventID 1000 

   [ Qualifiers]  0 
 
   Version 0 
 
   Level 2 
 
   Task 100 
 
   Opcode 0 
 
   Keywords 0x80000000000000 
 
  - TimeCreated 

   [ SystemTime]  2024-05-10T08:48:38.5512033Z 
 
   EventRecordID 235362 
 
   Correlation 
 
  - Execution 

   [ ProcessID]  0 
   [ ThreadID]  0 
 
   Channel Application 
 
   Computer my1 
 
   Security 
 

- EventData 

   LosslessCut.exe 
   3.59.1.0 
   651cb3d9 
   LosslessCut.exe 
   3.59.1.0 
   651cb3d9 
   80000003 
   0000000001b8193c 
   570 
   01daa2b6d8783c0a 
   Y:\winstuff\lossless-cut\LosslessCut.exe 
   Y:\winstuff\lossless-cut\LosslessCut.exe 
   1ceee4e2-39f5-49cd-a984-974a0b7a1039 
```

See attached image.
![Capture](https://github.com/mifi/lossless-cut/assets/76987202/bb93a068-aecf-4963-8032-1e929223f0d0)

Any help would be appreciated.  Thank you.

**Comment by mifi** [comment]
plase try to run it from a non-network drive. see also: #1893 #1903
